public/firefox.html

```diff
@@ -44,6 +44,7 @@ <h1 class="browser-name page-heading">{{ browser.name }}</h1>
             <li data-headset="htc_vive"><span>Save the <code>openvr_api.dll</code> file somewhere on your computer where the user running Firefox can read it (e.g., <code>c:\openvr\</code>).</span></li>
             <li data-headset="htc_vive"><span>In Firefox Nightly, navigate to <code>about:config</code>; change the value of <code>dom.vr.openvr.enabled</code> to <code>true</code> and <code>gfx.vr.openvr-runtime</code> to the full path of the <code>openvr_api.dll</code> file (e.g., <code>c:\openvr\openvr_api.dll</code>).</span></li>
             <li data-headset="oculus_rift"><span>Ensure that your Oculus Home settings allow for <a href="https://blog.mozvr.com/oculus-home-rift-cv1-webvr/#enablingunknownsources">Unknown Sources</a>.</span></li>
+            <li><span>On computers (especially laptops) with Nvidia chipsets, you may need to <a href="http://alteredqualia.com/texts/optimus/">enable the discrete GPU</a>.</span></li>
```

Comment:
As the maintainer, you should be able to also just edit my branch directly with the requested changes and merge if you prefer :-) Otherwise, I'll do it first thing tomorrow. Thanks!